src/components/Overlay.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Tags from './Tags'
+import { ReactComponent as Arrow } from '../icons/back-arrow.svg'
+import { ReactComponent as Share } from '../icons/share.svg'
+import Contributors from './Contributors'
+import Detail from './Detail'
+
+export default function Overlay({ project, onBack }) {
+  return (
+    <OverlaySection>
+      <Headline>
+        <ArrowStyled onClick={onBack} />
+        <Title>{project.title}</Title>
+        <ShareStyled />
+      </Headline>
+      <Img alt='' src={project.image ?? 'https://source.unsplash.com/random'}></Img>
+      <TagList>
+        {project.tags.map((tag, index) => (
+          <Tags content={tag} key={tag} />
+        ))}
+      </TagList>
+      <Contributors contributors={project.contributors} />
+      {project.details &&
+        project.details.map((detail) => (
+          <Detail title={detail.title} content={detail.information} />
+        ))}
+      <Description>{project.description}</Description>
+    </OverlaySection>
+  )
+}
+
+Overlay.propTypes = {
+  project: PropTypes.shape({
+    tags: PropTypes.array.isRequired,
+    country: PropTypes.string.isRequired,
+    description: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    image: PropTypes.string,
+    mission: PropTypes.string.isRequired,
+    about: PropTypes.string.isRequired,
+    updates: PropTypes.string.isRequired,
+    comments: PropTypes.string.isRequired,
+    contributors: PropTypes.arrayOf(
+      PropTypes.shape({
+        name: PropTypes.string.isRequired,
+        picture: PropTypes.string.isRequired,
+      })
+    ),
+  }),
+}
+const OverlaySection = styled.section`
+  position: fixed;
+  overflow-y: scroll;
+  top: 20px;
+  height: 100vh;
+  width: 100vw;
+  z-index: 99;
+  background-color: white;
+  border-radius: 10px 10px 0 0;
+  box-shadow: 0 -1px 10px rgba(46, 46, 58, 0.5);
+  padding: 10px;
+`
+const Img = styled.img`
+  margin-top: 50px;
+  width: 100%;
+  height: 250px;
```

Comment:
Fixed pixel values can be tricky on diffrent screen sizes. Maybe you can go for % as in your height?

Reply:
I will have a general design improvement story sometime soon where I will address these issues and the aforementioned ones. Thanks for your code review!